Pipfile

```diff
@@ -0,0 +1,22 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[packages]
+thoth-storages = "*"
+thoth-common = "*"
+prometheus-client = "*"
+
+[dev-packages]
+pytest = "*"
+pytest-timeout = "*"
+pytest-cov = "*"
+hypothesis-auto = "*"
+hypothesis = "*"
+pytest-mypy = "*"
+mypy = "*"
+pre-commit = "*"
```

Comment:
These dev dependencies are not used anywhere.

Reply:
mypy and pre-commit locally before sending PR, don't we keep them?

Reply:
They are not part of any testsuite. They are installed in CI, actually pre-installed in container images that the CI uses which gives one central control on their versions.

Reply:
Yes but locally we have to make the changes and therefore we need to have those libraries available to adjust them, CI just show the failure and where, but it is much faster to run it locally and have the changes. If the answer is to have them on local machine (which also has its own version not matching CI, then ok I ll do that :) )